packages/devtools_server/lib/src/file_system.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+class LocalFileSystem {
+  static String userHomeDir() {
+    final String envKey =
+        Platform.operatingSystem == 'windows' ? 'APPDATA' : 'HOME';
+    final String value = Platform.environment[envKey];
+    return value == null ? '.' : value;
+  }
+
+  static File file(String pathFromHomeDir) {
+    final file = File('${userHomeDir()}/$pathFromHomeDir');
+    if (!file.existsSync()) {
+      return null;
+    }
+    return file;
+  }
+
+  static String fileAsJson(String pathFromHomeDir) {
+    final _file = file(pathFromHomeDir);
```

Comment:
nit: don't use privates to avoid naming conflicts. Maybe it is a sign the method name is not good if it also makes a good variable name as methods should typically be verbs while variable names should be nouns.

Reply:
fixed